modules/ezcontent_node/modules/ezcontent_smart_article/src/Plugin/Field/FieldType/EntityReferenceSmarttags.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Drupal\ezcontent_smart_article\Plugin\Field\FieldType;
+
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\Core\TypedData\DataDefinition;
+use Drupal\Core\Field\Plugin\Field\FieldType\EntityReferenceItem;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Field\FieldItemBase;
+
+/**
+ * Plugin implementation of the 'ezcontent_smart_tags' field type.
+ *
+ * @FieldType(
+ *   id = "ezcontent_smart_tags",
+ *   label = @Translation("Entity reference w/smart tags"),
+ *   description = @Translation("Entity reference with smart tags"),
+ *   category = @Translation("Reference"),
+ *   default_widget = "ezcontent_smart_tags_autocomplete_tags",
+ *   default_formatter = "ezcontent_smart_tags_entity",
```

Comment:
Can we use `EntityReferenceEntityFormatter` ad get rid of file `modules/ezcontent_node/modules/ezcontent_smart_article/src/Plugin/Field/FieldFormatter/EntityReferenceSmarttagsEntityFormatter.php`

Reply:
Hey @vaibhavjain-in 
I checked this and we need to define a new custom formatter supporting our custom field type definition. so the  file `modules/ezcontent_node/modules/ezcontent_smart_article/src/Plugin/Field/FieldFormatter/EntityReferenceSmarttagsEntityFormatter.php` needs to stay.